STACpopulator/populator_base.py

```diff
@@ -57,7 +53,29 @@ def __init__(
         self.create_stac_collection()
 
     def load_config(self):
-        self._collection_info = load_collection_configuration()
+        """
+        Reads details of the STAC Collection to be created from a configuration file.
+
+        Once called, the collection information attribute should be set with relevant mapping attributes.
+        """
+        # use explicit override, or default to local definition
+        if not self._collection_config_path or not os.path.isfile(self._collection_config_path):
+            impl_path = inspect.getfile(self.__class__)
```

Comment:
Isn't this the path of `populator_base` ? but there is no `collection_config.yml` at that level.